tinygrad/tensor.py

```diff
@@ -158,6 +158,8 @@ def arange(stop, start=0, step=1, **kwargs): return Tensor(np.arange(start=start
   def where(self:Tensor, input_:Union[Tensor, float], other:Union[Tensor, float]):
     cond = (self != 0.0)
     return cond * input_ + (1.0 - cond) * other
+  
+  def numel(self): return functools.reduce(lambda x,y:x*y,self.shape)
```

Comment:
Can this not just be `math.prod(self.shape)`?

Reply:
yeah, sure. will change it.